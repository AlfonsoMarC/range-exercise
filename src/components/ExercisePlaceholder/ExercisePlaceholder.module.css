.container {
  display: flex;
  flex-direction: column;
  flex: 1;
  align-items: center;
  justify-content: center;
  padding: var(--size8);
  font-size: var(--font-size-m);
  color: var(--color-content);
}
