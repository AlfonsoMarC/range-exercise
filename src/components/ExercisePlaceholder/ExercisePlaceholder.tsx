import React from "react";
import styles from "./ExercisePlaceholder.module.css";

export interface ExercisePlaceholderProps {
  message: string;
}

const ExercisePlaceholder: React.FC<ExercisePlaceholderProps> = ({ message }) => {
  return <div className={`${styles.container} exercise-placeholder__container`}>{message}</div>;
};

export default ExercisePlaceholder;
