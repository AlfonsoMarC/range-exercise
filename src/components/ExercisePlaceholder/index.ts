import ExercisePlaceholder, { ExercisePlaceholderProps } from "./ExercisePlaceholder";
export type { ExercisePlaceholderProps };
export default ExercisePlaceholder;
