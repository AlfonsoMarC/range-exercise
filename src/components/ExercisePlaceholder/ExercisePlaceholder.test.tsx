import React from "react";
import { render } from "@testing-library/react";
import ExercisePlaceholder from "./ExercisePlaceholder";

describe("ExercisePlaceholder", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<ExercisePlaceholder message="Test Placeholder Message" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
