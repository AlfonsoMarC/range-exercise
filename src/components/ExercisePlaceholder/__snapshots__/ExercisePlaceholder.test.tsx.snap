// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExercisePlaceholder should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="container exercise-placeholder__container"
  >
    Test Placeholder Message
  </div>
</DocumentFragment>
`;
